components/homme/src/share/vertremap_base.F90

```diff
@@ -104,7 +104,7 @@ subroutine remap1(Qdp,nx,qsize,dp1,dp2)
      call remap1_nofilter(qdp,nx,qsize,dp1,dp2)
      return
   endif
-  if (vert_remap_q_alg == 1 .or. vert_remap_q_alg == 2 .or. vert_remap_q_alg == 3) then
+  if (vert_remap_q_alg == 1 .or. vert_remap_q_alg == 2 .or. vert_remap_q_alg == 3 .or. vert_remap_q_alg == 10) then
```

Comment:
Here it would be nice to error out if the q_alg value is not recognized. Right now the UKMO alg will silently run.